source/js/components/share-button-group/share-button-group.jsx

```diff
@@ -109,14 +109,13 @@ export default class ShareButtonGroup extends React.Component {
     });
 
     return (
-      <a
-        class={classes}
-        href="#"
+      <button
```

Comment:
e.g., 

https://foundation-s-issue-3366-qgqe9x.herokuapp.com/en/blog/initial-test-blog-post-with-fixed-title/
<img width="91" alt="image" src="https://user-images.githubusercontent.com/2896608/77009733-ac153980-6925-11ea-9b29-6f21ecaa8b88.png">
and further down the page
<img width="254" alt="image" src="https://user-images.githubusercontent.com/2896608/77009756-b8999200-6925-11ea-82d1-7f52fdecc171.png">